frontend/src/sockets/socketClient.js

```diff
@@ -0,0 +1,14 @@
+import {io} from 'socket.io-client';
```

Comment:
Should I change this to use export default?

Reply:
i personally think this is fine on the front end